src/main/java/carpetaddonsnotfound/CarpetAddonsNotFoundSettings.java

```diff
@@ -84,4 +95,18 @@ public enum ReplaceableFlowersOptions {
     ONE_TALL_FLOWERS(),
     ALL_FLOWERS()
   }
+
+
+  private static class OneHourMaxDelayLimit extends Validator<Integer> {
```

Comment:
Is their a validator class in carpet we could  reuse for this? Or is it also a private class in the Carpet settings server?

If not, I think we should add a sub `validators` folder/package in the `carpetaddonsnotfound` folder/package and include this class there in its own file. If we need to add more Validators in the future then they can go there.

Reply:
It's private in base carpet.